hearts/hearts.py

```diff
@@ -10,6 +10,12 @@ class Player(object):
     def __init__(self, username):
         self.username = username
 
+    def __eq__(self, other):
+        return self.username == other.username
+
+    def __hash__(self):
+        return hash(self.username)   # FIXME: Is this the best option?
```

Comment:
This is fine, because you can't have two players with the same username at the same table (it would be hard to visually distinguish them when you're playing the game!). There are ways to allow for duplicate usernames, but let's worry about that only if we need to.